src/api.js

```diff
@@ -9,28 +9,12 @@
  */
 var express = require('express'),
 	api = express(), 
-	token = process.env.GIT_TOKEN,
 	Promise = require('bluebird');
 
 // route the api module to host:port/api
 api.get('/users/:id/', function(req, res){
 	res.end('Not implemented.');
 });
 
-api.get('/github/status', function(req, res){
```

Comment:
requires re-implementation regarding session management and inclusion of save.js